### PR TITLE
[online-cmaq] Feature/nexus post fix

### PR DESCRIPTION
**Pull request by bbakernoaa**
## DESCRIPTION OF CHANGES: 
Update to the nexus_post job to properly combine and add in changes to accomidating the NEXUS_NUM_SPLIT environment variable.  When `NEXUS_NUM_SPLIT > 1` then an extra step is taken to combine the split nexus jobs into a single file.  More details here pertaining to the specific nexus utility scripts: https://github.com/noaa-oar-arl/NEXUS/pull/19

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## TESTS CONDUCTED: 

- [ ] hera.intel
- [ ] orion.intel
- [ ] cheyenne.intel
- [ ] cheyenne.gnu
- [ ] gaea.intel
- [ ] jet.intel
- [x] wcoss2.intel
- [ ] NOAA Cloud (indicate which platform)
- [ ] Jenkins
- [ ] fundamental test suite
- [ ] comprehensive tests (specify *which* if a subset was used)

## DEPENDENCIES:

- [ ] My code follows the style guidelines in the Contributor's Guide
- [x] I have performed a self-review of my own code using the Code Reviewer's Guide
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes need updates to the documentation. I have made corresponding changes to the documentation
- [x] My changes do not require updates to the documentation (explain).
- [x] My changes generate no new warnings
- [x] New and existing tests pass with my changes
- [ ] Any dependent changes have been merged and published

## LABELS (optional): 

A Code Manager needs to add the following labels to this PR: 
- [ ] Work In Progress
- [ ] bug
- [ ] enhancement
- [ ] documentation
- [ ] release
- [ ] high priority
- [ ] run_ci
- [ ] run_we2e_fundamental_tests
- [ ] run_we2e_comprehensive_tests
- [ ] Needs Cheyenne test 
- [ ] Needs Jet test 
- [ ] Needs Hera test 
- [ ] Needs Orion test 
- [ ] help wanted

## CONTRIBUTORS (optional): 
@zmoon @bbakernoaa

### CODE REVIEW

File: scripts/exregional_nexus_post_split.sh [hunk 1]
```diff
@@ -7,8 +7,8 @@
 #
 #-----------------------------------------------------------------------
 #
+. ${GLOBAL_VAR_DEFNS_FP}
 . $USHdir/source_util_funcs.sh
-source_config_for_task "cpl_aqm_parm" ${GLOBAL_VAR_DEFNS_FP}
```
Comment (zmoon) [inline scripts/exregional_nexus_post_split.sh]: does the removed line not work? seems like it should be doing the same thing?

**Comment by bbakernoaa** [comment]
@chan-hoo I think this is ready for a test to be pulled.  We will also need to update the hash on NEXUS ([8b5ef7e](https://github.com/noaa-oar-arl/NEXUS/commit/8b5ef7e995b7cd54e5f31f755feb770b4eedb766)) as there is a small utility to merge the N-split NEXUS files together into a single file

**Comment by chan-hoo** [comment]
@bbakernoaa, Can you 1) update your branch with the latest 'online-cmaq' branch, 2) change the hash of NEXUS to "8b5ef7e" in "externals/Externals_AQM.cfg", and 3) change the status from "Draft" to "Ready for review"?

File: scripts/exregional_nexus_post_split.sh [hunk 2]
```diff
@@ -7,7 +7,7 @@
 #
 #-----------------------------------------------------------------------
 #
-. $USHdir/source_util_funcs.sh
+. ${GLOBAL_VAR_DEFNS_FP}
```
Comment (chan-hoo) [inline scripts/exregional_nexus_post_split.sh]: @bbakernoaa, can you remove this change? It was a recent update in the 'develop' branch.

File: scripts/exregional_nexus_post_split.sh [hunk 3]
```diff
@@ -125,6 +142,7 @@ cp_vrfy ${ARL_NEXUS_DIR}/utils/run_nco_combine_ant_bio.sh .
 #
 #-----------------------------------------------------------------------
 #
+echo ${INPUT_DATA}
```
Comment (chan-hoo) [inline scripts/exregional_nexus_post_split.sh]: @bbakernoaa, please remove this line.

**Comment by chan-hoo** [comment]
@bbakernoaa, the test was completed successfully. If you want to merge this PR, please change to "Ready for review".

**Review by chan-hoo — APPROVED** [review]

**Pull request merged**